client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/ContactDateCard.tsx

```diff
@@ -8,6 +8,7 @@ import Interaction from 'models/Contexts/InteractionEventDialogData';
 import PrimaryButton from 'commons/Buttons/PrimaryButton/PrimaryButton';
 
 import useStyles from './ContactDateCardStyles';
+import useFormStyles from 'styles/formStyles';
```

Comment:
imports

Reply:
whats the problem here? 😅

Reply:
import useFormStyles from 'styles/formStyles'; - in line 9